__tests__/reducers/ui.spec.js

```diff
@@ -31,12 +38,13 @@ describe( 'notification reducer', () =>
         {
             expect(
                 ui(
-                    {},
+                    { windows: [ { windowId: 1, settingsMenuIsVisible: true } ] },
                     {
-                        type : TYPES.HIDE_SETTINGS_MENU
+                        type    : TYPES.HIDE_SETTINGS_MENU,
+                        payload : { windowId: 1 }
                     }
                 )
-            ).toEqual( { settingsMenuIsVisible: false } );
+            ).toEqual( uiStateHide );
         } );
```

Comment:
@manavbp we're missing tests for add/remove window here.

Also we need tests for the added actions too. 👍